acl/user.go

```diff
@@ -0,0 +1,14 @@
+package acl
+
+type User interface {
```

Comment:
nah, root is a special role. the root user is the only one that has this static role (for now)